docs/developers-guide/files/spark-backend.md

```diff
@@ -200,6 +200,31 @@ For example, the three Elm snippets above are translated into this Spark code:
     ))
 ```
 
+## Aggregations
+The current Spark API recognizes the following patterns as aggregations:
+
+```
+source
+    |> List.map mappingFunction
+    |> (\a ->
+        [{
+            label = aggregationFunction a
+        }]
+    )
```

Comment:
This comment applies to minimum in PR #804 as well.  The idiomatic way to write this in Elm is:

``` Elm
source
  |> List.map mappingFunction
  |> List.maximum
```

We want to support the same so that the modeler of the logic is blissfully unaware that Spark is involved. It looks like that means we'll have to manage around Spark, for example, by defaulting "label" to something.